pkg/kubelet/apis/kubeletconfig/fuzzer/fuzzer.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fuzzer
+
+import (
+	"path/filepath"
+	"time"
+
+	"github.com/google/gofuzz"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	runtimeserializer "k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/kubernetes/pkg/kubelet/apis/kubeletconfig"
+	"k8s.io/kubernetes/pkg/kubelet/apis/kubeletconfig/v1alpha1"
+	"k8s.io/kubernetes/pkg/kubelet/qos"
+	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
+	"k8s.io/kubernetes/pkg/master/ports"
+)
+
+// Funcs returns the fuzzer functions for the kubeadm apis.
+func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
```

Comment:
@mtaufen I guess you need to register `kubeletconfigfuzzer.Funcs`

https://github.com/kubernetes/kubernetes/blob/dd9981d038012c120525c9e6df98b3beb3ef19e1/pkg/api/testing/fuzzer.go#L92-L107

Reply:
What does registering fuzzer funcs here do, vs just passing them into a call to RoundTripTestForAPIGroup?

Reply:
I'm not sure about it. Just seeing other fuzzers register here so I guess the newly added fuzzer also needs to register :)

Reply:
I tested locally, whether it is registered here or not, the test can pass...
@sttts may have some contexts. Thanks!

Reply:
Please only add fuzzer for apiserver types here, not for configs only used locally. Look for roundtrip_test.go files in the code-base. There are some more which have local fuzzers.

Reply:
@sttts thanks for clarifying!